src/main/java/com/org/trustservice/service/TrustGroupService.java

```diff
@@ -0,0 +1,204 @@
+package com.org.trustservice.service;
+
+import com.org.trustservice.dto.DepartmentTrustGroupsDTO;
+import com.org.trustservice.dto.TrustGroupDTO;
+import com.org.trustservice.dto.TrustGroupUpdateDTO;
+import com.org.trustservice.dto.TrustGroupUpdateResponseDTO;
+import com.org.trustservice.excpetion.DatabaseException;
+import com.org.trustservice.excpetion.ResourceNotFoundException;
+import com.org.trustservice.model.DepartmentTrustGroup;
+import com.org.trustservice.model.TrustGroupFlavour;
+import com.org.trustservice.model.TrustGroupPermission;
+import com.org.trustservice.repository.DepartmentTrustGroupRepository;
+import com.org.trustservice.repository.PermissionRepository;
+import com.org.trustservice.repository.TrustGroupFlavourRepository;
+import com.org.trustservice.util.TrustGroupNameComparator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+/**
+ * @author
+ */
+@Service
+public class TrustGroupService {
+
+
+    @Autowired
+    DepartmentTrustGroupRepository departmentTGRepository;
+
+    @Autowired
+    TrustGroupFlavourRepository trustGroupFlavourRepository;
+
+    @Autowired
+    PermissionRepository permissionRepository;
+
+    @Value("${google.permissions.1}")
```

Comment:
whenever you have 3 or more values coming from same base type, like `google.permissions.*` put them in a POJO class.
https://www.baeldung.com/configuration-properties-in-spring-boot